app/assets/stylesheets/bento.scss

```diff
@@ -376,3 +378,28 @@ $z-depth-way-front:     1000;
 .about-bento {
   margin: 4rem 0;
 }
+
+.title-area {
+  margin-top: 1.5rem;
+  margin-bottom: 1rem;
+}
+
+.title.has-subline {
+  margin-bottom: .1em;
+  padding-bottom: .1em;
+}
+
+.subline {
+  font-size: $fs-xsmall;
+}
+
+.nav .current {
+  font-weight: $fw-bold;
+}
+
+// changes for testing - TODO: remove
```

Comment:
I sorta want to hide this by default and have a debug mode that turns it back on. But we can do that later.